sdk_container/src/third_party/coreos-overlay/app-emulation/amazon-ssm-agent/amazon-ssm-agent-3.2.985.0-r2.ebuild

```diff
@@ -1,57 +0,0 @@
-# Distributed under the terms of the GNU General Public License v2
-
-EAPI=7
-
-COREOS_GO_PACKAGE="${GITHUB_URI}"
-COREOS_GO_VERSION="go1.19"
-
-inherit coreos-go-depend golang-vcs-snapshot systemd
-
-EGO_PN="github.com/aws/${PN}"
-DESCRIPTION="AWS Systems Manager Agent"
-HOMEPAGE="https://github.com/aws/amazon-ssm-agent"
-LICENSE="Apache-2.0"
-SRC_URI="https://${EGO_PN}/archive/${PV}.tar.gz -> ${P}.tar.gz ${EGO_VENDOR_URI}"
-SLOT="0"
-KEYWORDS="amd64 arm64"
-
-S="${WORKDIR}/${PN}-${PV}/src/${EGO_PN}"
-
-PATCHES=(
-)
-
-src_prepare() {
-	default
-	ln -s ${PWD}/vendor/src/* ${PWD}/vendor/
-}
-
-src_compile() {
-	go_export
-
-	# this is replication of commands from the vendor makefile
-	# but without network activity during build phase
```

Comment:
The reason for the duplication was the network access, is this not the case anymore or if it is the case, are we fine with network access? We could also patch the Makefile instead of duplicating the commands.